client/packages/executor/src/executionManager/sideEffect.ts

```diff
@@ -327,6 +335,8 @@ export class SideEffect extends EventEmitter {
         this.txStatus = TxStatus.Pending // acts as mutex lock
         this.minProfitUsd = this.strategyEngine.getMinProfitUsd(this)
         const bidUsd = this.biddingEngine.computeBid(this)
+        // TODO: should this be here or in "processBid"?
```

Comment:
I would add this to processBid I think, then we ensure one source of these, the incoming events